addons/website/static/lib/summernote/src/js/EventHandler.js

```diff
@@ -420,7 +420,7 @@ define([
           $btn.parents('.popover').hide();
         }
 
-        if ($.isFunction($.summernote.pluginEvents[eventName])) {
+        if ($.isFunction($.summernote.pluginEvents[eventName]) && eventName != "formatBlock") {
```

Comment:
I compare <i>SummerNote master</i> and <i>Odoo SummerNote</li> lot's of override or changes are apply on library files,

View following screenshot,
![summernote_eventhandler_js_cropped](https://cloud.githubusercontent.com/assets/7599891/6014870/6a88d770-ab92-11e4-8f4f-eba28327a00b.png)

This <i>if....else..</i> condition block portion copy from summernote master to our library <i>formatblock</i> functionality worked, but <i>orderlist</i> and <i>unorderlist</i> functionality are break. Can you suggest what can I do for this case. thank you,